awx/main/models/jobs.py

```diff
@@ -505,6 +487,10 @@ class Meta:
         on_delete=models.SET_NULL,
         help_text=_('The SCM Refresh task used to make sure the playbooks were available for the job run'),
     )
+    credentials = models.ManyToManyField(
```

Comment:
@AlanCoding `Job.credentials` is moved here (not `JobOptions`, which `JT` shares)

Reply:
I thought that it would go on the UnifiedJob model.

Reply:
👍 testing now